### ISSUE TITLE
Bad markdown highlight inside words

### ISSUE THREAD

**Issue by roipoussiere** [shd101wyy/markdown-preview-enhanced#320]
When using blod (`**`) and emphasis (`*`) inside words, the highlight do not stop.

### code

```
**foo**bar

foo
```

### actual highlight
---
**foobar**

**foo**

---

### expected hightlight

---
**foo**bar

foo

---

### code

```
*foo*bar

foo
```

### actual highlight
---
*foobar*

*foo*

---

### expected hightlight

---
*foo*bar

foo

---

Note that this is correctly rendered in the right pane.

**Comment by shd101wyy** [comment]
It is [language-gfm](https://github.com/atom/language-gfm) issue.  
This package only deals with preview not the editor.  
Thanks

**Comment by roipoussiere** [comment]
Yep and already reported [here](https://github.com/atom/language-gfm/issues/117).
